Storybook doesn't run on master

After merging https://github.com/magmo/apps/pull/368 (at least, I'm assuming this is the culprit) storybook won't run on master (for me).

The error that storybook fails with is:
```
ERROR in ./src/utils/contract-utils.ts
Module not found: Error: Can't resolve '../../build/contracts/ConsensusApp.json'
```
When I try doing a `yarn start` to build the contract, that fails with:
```
DATA:  Compiling ./contracts/Consensus.sol...

DATA:
Compilation warnings encountered:

// loads of warnings ...

DATA:  Error: Returned values aren't valid, did it run Out of Gas?
```

I'm using the following temporary fix on my branch to get storybook to run again: https://github.com/magmo/apps/commit/01b972c095d5f0c9cec4932c5deee8d1ac20363c.

--- a/packages/wallet/package.json
+++ b/packages/wallet/package.json
@@ -21,7 +21,7 @@
     "postinstall": "rm -f node_modules/web3/index.d.ts",
     "storybook": "start-storybook -p 9001 -c .storybook",
     "test:contracts": "npx ganache-then-jest -c ./config/jest/jest.contracts.config.js --runInBand",
-    "": "rm -rf ./build/contracts"
+    "clearContracts": "rm -rf ./build/contracts"
   },
   "dependencies": {
     "@firebase/app-types": "^0.3.2",

--- a/packages/wallet/package.json
+++ b/packages/wallet/package.json
@@ -12,7 +12,7 @@
     "start": "npm run clearContracts && node scripts/start.js",
     "build": "npm run clearContracts && node scripts/build.js",
     "test": "run-s prettier:check 'test:app --all'",
-    "test:ci": "DEV_GANACHE_PORT=8503 clearContracts run-s prettier:check build 'test:contracts --all --ci' 'test:app --all --ci --runInBand'",
+    "test:ci": "DEV_GANACHE_PORT=8503  run-s clearContracts prettier:check build 'test:contracts --all --ci' 'test:app --all --ci --runInBand'",
     "test:app": "npx run-jest -c ./config/jest/jest.config.js",
     "prettier:check": "npx prettier --check 'src/**/*.{ts,tsx}'",
     "prettier:write": "npx prettier --write 'src/**/*.{ts,tsx}'",
@@ -21,7 +21,7 @@
     "postinstall": "rm -f node_modules/web3/index.d.ts",
     "storybook": "start-storybook -p 9001 -c .storybook",
     "test:contracts": "npx ganache-then-jest -c ./config/jest/jest.contracts.config.js --runInBand",
-    "clearContracts": "rm -rf ./build/contracts"
+    "": "rm -rf ./build/contracts"
   },
   "dependencies": {
     "@firebase/app-types": "^0.3.2",

--- a/packages/wallet/package.json
+++ b/packages/wallet/package.json
@@ -9,10 +9,10 @@
     "firebase/**/grpc": "^1.16.1"
   },
   "scripts": {
-    "start": "node scripts/start.js",
-    "build": "node scripts/build.js",
+    "start": "npm run clearContracts && node scripts/start.js",
+    "build": "npm run clearContracts && node scripts/build.js",
     "test": "run-s prettier:check 'test:app --all'",
-    "test:ci": "DEV_GANACHE_PORT=8503 run-s prettier:check build 'test:contracts --all --ci' 'test:app --all --ci --runInBand'",
+    "test:ci": "DEV_GANACHE_PORT=8503 clearContracts run-s prettier:check build 'test:contracts --all --ci' 'test:app --all --ci --runInBand'",
     "test:app": "npx run-jest -c ./config/jest/jest.config.js",
     "prettier:check": "npx prettier --check 'src/**/*.{ts,tsx}'",
     "prettier:write": "npx prettier --write 'src/**/*.{ts,tsx}'",
@@ -20,7 +20,8 @@
     "ganache:start": "npx start-ganache",
     "postinstall": "rm -f node_modules/web3/index.d.ts",
     "storybook": "start-storybook -p 9001 -c .storybook",
-    "test:contracts": "npx ganache-then-jest -c ./config/jest/jest.contracts.config.js --runInBand"
+    "test:contracts": "npx ganache-then-jest -c ./config/jest/jest.contracts.config.js --runInBand",
+    "clearContracts": "rm -rf ./build/contracts"
   },
   "dependencies": {
     "@firebase/app-types": "^0.3.2",

